packages/mobile/src/tokens/utils.test.tsx

```diff
@@ -62,3 +73,39 @@ describe(getHigherBalanceCurrency, () => {
     ).toEqual(undefined)
   })
 })
+
+describe('token to fiat exchanges', () => {
+  function TestComponent({ tokenAddress }: { tokenAddress: string }) {
+    const isCoreToken = useIsCoreToken(tokenAddress)
+
+    return (
+      <View>
+        <Text testID="isCoreToken">{isCoreToken}</Text>
+      </View>
+    )
+  }
+
+  it('cUSD, cEUR and CELO are core tokens', async () => {
```

Comment:
Why can't you just check like this?
```
[mockCusdAddress, mockCeurAddress, mockCeloAddress].each((token) => expect(useIsCoreToken(token).toBeTruthy())
```

Reply:
Oh, I see, is it because you need the `store={createMockStore()}`?
Can't we just mock `useSelector(tokensListSelector)`? 
I mean, it's ok but it seems a lot of extra code to test `useIsCoreToken`

Reply:
It's because testing hooks is a PITA :( I agree it's not great.

You can't just call a hook, you must do it inside a React component. So the only way to test a hook (that I found) is to create a fake component and print the result in the screen

Reply:
I deleted the hook so these tests don't exist anymore :)